source/common/http/conn_manager_impl.h

```diff
@@ -60,7 +61,12 @@ namespace Http {
   COUNTER(downstream_rq_4xx)                                                                       \
   COUNTER(downstream_rq_5xx)                                                                       \
   TIMER  (downstream_rq_time)                                                                      \
-  COUNTER(failed_generate_uuid)
+  COUNTER(failed_generate_uuid)                                                                    \
+  COUNTER(tracing_random_sampling)                                                                 \
```

Comment:
why did we move tracing stats here vs. where they were before? It seems like we should only have connection manager tracing stats if tracing is configured under the "http.<name>.tracing." namespace?